compiler/rustc_errors/src/emitter.rs

```diff
@@ -1716,15 +1716,8 @@ impl EmitterWriter {
             // Instead of using .to_string().len(), we iteratively count the
             // number of digits to avoid allocation. This strategy has sizable
             // performance gains over the old string strategy.
```

Comment:
This comment can be removed, if you moved calculation into separate function?

Reply:
Right. Done.